background/constants/networks.ts

```diff
@@ -494,19 +299,223 @@ export const DEFAULT_QUAI_LOCAL = {
   ]
 } as Network
 
-export const DEFAULT_QUAI_NETWORKS = [
-  DEFAULT_QUAI_TESTNET,
-  DEFAULT_QUAI_DEVNET,
-  DEFAULT_QUAI_LOCAL
+export const QUAI_NETWORK: EVMNetwork = {
+  name: "Quai Network", // maybe add "Colosseum"
+  baseAsset: QUAI,
+  chainID: "9000",
+  family: "EVM",
+  chains: DEFAULT_QUAI_TESTNET.chains,
+  isQuai: true,
+}
+
+export const QUAI_NETWORK_LOCAL: EVMNetwork = {
+  name: "Quai Network Local",
+  baseAsset: QUAI_LOCAL,
+  chainID: "1337",
+  family: "EVM",
+  chains: DEFAULT_QUAI_LOCAL.chains,
+  isQuai: true,
+}
+
+
+export const DEFAULT_NETWORKS = [
+  QUAI_NETWORK,
+  QUAI_NETWORK_LOCAL,
+  /*ETHEREUM,
+  POLYGON,
+  OPTIMISM,
+  GOERLI,
+  ARBITRUM_ONE,
+  ROOTSTOCK,
+  AVALANCHE,
+  BINANCE_SMART_CHAIN,
+  ...wrapIfEnabled(FeatureFlags.SUPPORT_ARBITRUM_NOVA, ARBITRUM_NOVA),*/
 ]
 
-export class QuaiContext {
+export function isBuiltInNetwork(network: EVMNetwork): boolean {
+  return DEFAULT_NETWORKS.some(
+    (builtInNetwork) => builtInNetwork.chainID === network.chainID
+  )
+}
+
+export const DEFAULT_NETWORKS_BY_CHAIN_ID = new Set(
+  DEFAULT_NETWORKS.map((network) => network.chainID)
+)
+
+export const FORK: EVMNetwork = {
+  name: "Ethereum",
+  baseAsset: ETH,
+  chainID: process.env.MAINNET_FORK_CHAIN_ID ?? "69",
+  family: "EVM",
+  coingeckoPlatformID: "ethereum",
+}
+
+export const EIP_1559_COMPLIANT_CHAIN_IDS = new Set(
+  [ETHEREUM, POLYGON, GOERLI, AVALANCHE, QUAI_NETWORK, QUAI_NETWORK_LOCAL].map((network) => network.chainID)
+)
+
+export const CHAINS_WITH_MEMPOOL = new Set(
+  [ETHEREUM, POLYGON, AVALANCHE, GOERLI, BINANCE_SMART_CHAIN, QUAI_NETWORK, QUAI_NETWORK_LOCAL].map(
+    (network) => network.chainID
+  )
+)
+
+export const NETWORK_BY_CHAIN_ID = {
+  [QUAI_NETWORK.chainID]: QUAI_NETWORK,
+  [QUAI_NETWORK_LOCAL.chainID]: QUAI_NETWORK_LOCAL,
+  /*[ETHEREUM.chainID]: ETHEREUM,
```

Comment:
Lets delete these instead of just commenting them

Reply:
Deleted